app/assets/javascripts/spree/frontend/solidus_paypal_commerce_platform/button_actions.js

```diff
@@ -14,20 +14,7 @@ SolidusPaypalCommercePlatform.postOrder = function(payment_method_id) {
 
 SolidusPaypalCommercePlatform.approveOrder = function(data, actions) {
   actions.order.get().then(function(res){
-    var updated_address = res.purchase_units[0].shipping.address
-    Spree.ajax({
-      url: '/solidus_paypal_commerce_platform/update_address',
-      method: 'POST',
-      data: {
-        address: updated_address,
-        order_id: Spree.current_order_id,
-        order_token: Spree.current_order_token
-      },
-      error: function(response) {
-        message = response.responseJSON;
-        alert('There were some problems with your payment address - ' + message);
-      }
-    }).then(function() {
+    SolidusPaypalCommercePlatform.updateAddress(res).then(function() {
```

Comment:
Does `res` stand for `response`? 
Asking for a friend 😅

Reply:
Yes! PayPal uses `res` as shorthand for `response` in their documentation. I'm not sure if that's a good enough reason to use it here, maybe `response` would be a bit clearer.